tests/libnet/skips.go

```diff
@@ -14,3 +17,13 @@ func SkipWhenNotDualStackCluster(virtClient kubecli.KubevirtClient) {
 		Skip("This test requires a dual stack network config.")
 	}
 }
+
+func SkipWhenClusterHasASingleNode(virtClient kubecli.KubevirtClient) {
+	nodes, err := virtClient.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: v1.NodeSchedulable + "=" + "true"})
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "Should list compute nodes")
+	ExpectWithOffset(1, nodes.Items).ToNot(BeEmpty(), "There *must* be some compute node")
+
+	if len(nodes.Items) < 2 {
+		Skip("Migration tests require at least 2 nodes")
```

Comment:
The "Migration" is one potential client of this, at least per the skip name.
If you want to make it migration specific, then name the skip function accordingly, otherwise name this skip reason accordingly. You could pass the reasoning through a function argument and use a default if nothing was passed.

Reply:
Keeping it simple, and leaving it generic.